test/dish_test.rb

```diff
@@ -0,0 +1,32 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/dish'
+
+class DishTest < Minitest::Test
+
+  def test_it_exists
+
+    dish = Dish.new("Couscous Salad", :appetizer)
+    assert_instance_of Dish, dish
+  end
+
+  def test_for_new_dish
+
+    dish = Dish.new("Couscous Salad", :appetizer)
+
+    expected = "Couscous Salad"
+    actual = dish.name
+
+    assert_equal expected, actual
+  end
+
+  def test_for_dish_category
```

Comment:
These tests look really good, but make sure you are testing for more names and more categories - right now, this is only working because the way you have set up your dish will make everything an appetizer.